db/migrate/20181203003146_create_users.rb

```diff
@@ -0,0 +1,17 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      t.string   :name
+      t.string   :email
+      t.string   :encrypted_password
+      t.string   :battle_tag
+      t.string   :hotslogs_id
+      t.string   :site_role
+      t.boolean  :archived, default: false
+      t.datetime :archived_at
+      t.integer  :mmr
```

Comment:
will there be additional mmr types or is leadership leaning towards single number?
I could see hotslogs mmr, heroes profile mmr, hotlogs TL mmr, etc.

Reply:
I envision that we'd have a task to check hotslogs, and we'd write each individual mmr to its own key in users.rank. The average number (however we choose to calculate) would be stored in the users.mmr field.

I've made a json column to house all of the non-standard (and possibly changing, from time to time) values.
Something like this :
```
rank: [
  :hotslogs_hl,,
  :hotslogs_tl,
  :hotslogs_ur,
  :random_other_stats_site
]
```

Reply:
Perfect, you both beat me to this. Was hoping that was what the `jsonb` column was for. 😍